crates/matrix-sdk/src/oidc/mod.rs

```diff
@@ -1225,3 +1267,110 @@ fn hash<T: Hash>(x: &T) -> u64 {
     x.hash(&mut hasher);
     hasher.finish()
 }
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use mas_oidc_client::types::{
+        client_credentials::ClientCredentials, registration::ClientMetadata,
+    };
+    use matrix_sdk_base::SessionMeta;
+    use matrix_sdk_test::async_test;
+    use ruma::{
+        api::client::discovery::discover_homeserver::AuthenticationServerInfo, OwnedUserId,
+    };
+    use url::Url;
+
+    use crate::{
+        oidc::{
+            FullSession, OidcAccountManagementAction, RegisteredClientData, SessionTokens,
+            UserSession,
+        },
+        ClientBuilder,
+    };
+
+    #[async_test]
+    async fn test_account_management_url() {
```

Comment:
I'm totally assuming that the below doesn't make an requests to `example.com`?

Reply:
No indeed.